projects/baby_driver/scripts/can_message.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+# coding: utf-8
+"""python can message with 8 bytes"""
+from __future__ import print_function
+
+import can
+
+def send_message(msg):
+    """sends can message with 8 bytes. You can change the 2nd one"""
+    #change the channel later to can0 or vcan0
+    bus = can.interface.Bus(bustype='socketcan', channel='can0', bitrate=250000)
```

Comment:
The bitrate here needs to agree with the bitrate in the CAN settings, so change it to 500000. Also did you test this with vcan0 and the CAN message showed up on candump?

Reply:
Ah I did. Weird, it still received it.